recipes/scalene/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "scalene" %}
+{% set version = "1.3.16" %}
```

Comment:
Can this be updated? Currently, the latest is `v1.5.9.1`.

Reply:
Maybe this will help fix the build...

Reply:
My mistake. [The PyPI version](https://pypi.org/project/scalene/1.5.9/) does not match [the GitHub version](https://github.com/plasma-umass/scalene/releases/tag/v1.5.9.1).